### PR TITLE
mdigi.tools

### PR DESCRIPTION
**Pull request by CrispyMiner**
(I am sole focusing on the color tools on this site, but they don't have to the own specific color tool to pick from
Tried to make a change to fix color tiles from getting inverted and I need help trying to get the .color-shade from getting inverted (the lighter colors are prominently being changed, while the darker colors are only slightly).

The link to the specific tool in questions is 
https://mdigi.tools/color-shades/

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -8764,6 +8764,18 @@ section[class$="container"][style^="background-image"] {
 
 ================================
 
+mdigi.tools
+
+CSS
+.color-shade {
+    --darkreader-inline-bgcolor: none;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8772]: Not needed

Comment (CrispyMiner) [reply src/config/dynamic-theme-fixes.config:8772]: > Not needed

how would i get the colors do not have their saturation inverted then, for [Color Shades](https://mdigi.tools/color-shades/) specifically

Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8772]:
```suggestion
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -8764,6 +8764,18 @@ section[class$="container"][style^="background-image"] {
...
+
+IGNORE INLINE STYLE
+.color-preview *
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8775]:
```suggestion
.color-preview *
.color-shade
```

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -8764,6 +8764,27 @@ section[class$="container"][style^="background-image"] {
...
+
+CSS
+input {
+    color: #6B6B6B !important;
+}
+label, button, path,
+h1, h2, h3, h4, p, a,
+.site-title {
+    color: #808080 !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8777]: Hard no, where is this needed?

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -8764,6 +8764,27 @@ section[class$="container"][style^="background-image"] {
...
+[class*="ml-"] {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8778]: This targets a lot of elements, is this really needed as well?

Comment (CrispyMiner) [reply src/config/dynamic-theme-fixes.config:8778]: > This targets a lot of elements, is this really needed as well?

Yes, all of it is just text.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:8778]: Let me rephrase the question, why is this fix needed?
